unittests/test_cyl_lat_sph.py

```diff
@@ -2,175 +2,158 @@
 # test_cyl_lat_sph.py: conversions between rec, cyl, lat, and sph coordinates
 ##########################################################################################
 
-import os
 import numpy as np
-import unittest
+import pytest
+from pytest import approx
 from cspyce import *
+from pathlib import Path
 
-KERNELS_ = os.path.split(__file__)[0] + '/../unittest_support/'
-DPR = dpr()
-TWOPI = twopi()
+DPR = 180 / np.pi
+PI = np.pi
+TWOPI = 2 * np.pi
 
-class Test_cyl_lat_sph(unittest.TestCase):
 
-  def setUp(self):
-    furnsh(KERNELS_ + 'de421.bsp')
-    furnsh(KERNELS_ + 'naif0012.tls')
+@pytest.fixture(autouse=True)
+def load_kernel():
+    kernels = Path(__file__).parent.parent / 'unittest_support/'
```

Comment:
Should this have a trailing slash? Doesn't Path take care of that for you?

Reply:
Fixed.